src/main/java/org/mule/tools/apikit/model/APIFactory.java

```diff
@@ -103,13 +103,17 @@ private HttpListenerConnection buildDefaultHttpListenerConnection(String port) {
         .build();
   }
 
+  private String getUriLastSegment(String ramlPathUri) {
+    return ramlPathUri.substring(ramlPathUri.lastIndexOf("/") + 1);
```

Comment:
will this work for windows? also, add a test

Reply:
this is for windows,, current tests has coverage for this and we don't has test environment for windows cases.